### PR TITLE
[DH-394] fix publichealth

### PR DESCRIPTION
**Pull request by felder**
https://jira-secure.berkeley.edu/browse/DH-394

### CODE REVIEW

**Comment by shaneknapp** [comment]
i'm gonna build this locally and see what happens... :)

**Review by shaneknapp — APPROVED** [review]
this look good, and builds successfully, but due to the scope of the changes i'd feel best if @balajialg has some notebooks to test on staging before we merge to prod.

**Comment by balajialg** [comment]
okay, let me do some testing in the staging hub and report back!

**Comment by shaneknapp** [comment]
> okay, let me do some testing in the staging hub and report back!

btw, this hasn't been deployed to `staging` yet

File: Dockerfile [hunk 1]
```diff
@@ -54,15 +54,15 @@ RUN apt-get update > /dev/null && \
 COPY install-mambaforge.bash /tmp/install-mambaforge.bash
 RUN /tmp/install-mambaforge.bash
 
+RUN chown -Rh rstudio:rstudio /home/rstudio
+RUN chown -Rh rstudio:rstudio /srv/conda
+
```
Comment (ryanlovett) [inline Dockerfile:59]: If the /srv/conda chown is necessary, should the mambaforge install just be performed as NB_USER ? Then you could skip the recursive chown.

I'm surprised that it is necessary to chown /home/rstudio though.

**Comment by shaneknapp** [comment]
@balajialg if the changes look good to you, approve the PR and then merge.  after it builds and pushes a commit to the datahub repo, we can merge that and you can test on `staging`

**Review by balajialg — APPROVED** [review]
I will do some testing for PH 142, 251 and 253 notebooks when the changes gets merged to staging. Thanks for the heads up @shaneknapp !

File: Dockerfile (hunk 1 above)
Comment (felder) [reply Dockerfile:59]: The mambaforge install actually performs a chown.  I did s/rstudio/${NB_USER} also swapped /home/rstudio for ${HOME}

**Comment by shaneknapp** [comment]
LGTM!  feel free to merge

**Comment by felder** [comment]
Issue https://github.com/berkeley-dsep-infra/datahub/issues/6349

**Pull request merged**